tmt/trying.py

```diff
@@ -73,11 +73,11 @@ def menu(self) -> str:
         return before + key + after + padding + self.description
 
     @classmethod
-    def find(cls, key: str) -> "Action":
-        """ Return action for given keyboard shortcut """
+    def find(cls, answer: str) -> "Action":
+        """ Return action for given keyboard input (shortcut or whole word) """
 
         for action in cls:
-            if action.key == key:
+            if answer in (action.key, action.action):
```

Comment:
Would it make sense to ignore lower/upper case, i.e. allowing input like `tEsT` or `T` besides `test` and `t`? Using `action.lower()` instead of `action` would be a simple addition - seems like a good thing UX-wise, should we be so tolerant?

Reply:
I don't see a reason why not, but I'm a frequent “enjoyer” of `:Wq` :D so I may be biased with regards to this :)

Reply:
Added in [f5d19df](https://github.com/teemtee/tmt/pull/2691/commits/f5d19df8fc84016594c8eb44fcfb48ef6dbee49b)